rubi/logging_config.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from io import StringIO
+import requests
+import sys
+
+# Create an in-memory stream to store logs
+log_stream = StringIO()
+
+# flag to enable API endpoint
+api_enabled = True
+
+
+class APILogHandler(logging.Handler):
```

Comment:
Currently the api_enabled flag is hardcoded to true but this implementation currently prints to the Flask dashboard and to the command line